README.md

```diff
@@ -6,7 +6,7 @@ It's auto-generated by sonata-project/dev-kit package.
 
 # SonataDoctrineOrmAdminBundle
 
-Integrate Doctrine ORM into the SonataAdminBundle
+Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
```

Comment:
Where this change comes from?

Reply:
Dont know why but some changes like this happen sometimes. 
And then, they go away: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1240/files